Manifests/ManagedPreferencesApplications/com.twingate.macos.plist

```diff
@@ -0,0 +1,192 @@
+			<key>pfm_description_reference</key>
+			<string>A globally unique identifier for the payload. The actual content is unimportant, but it must be globally unique. In macOS, you can use uuidgen to generate reasonable UUIDs.</string>
+			<key>pfm_format</key>
+			<string>^[0-9A-Za-z]{8}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{4}-[0-9A-Za-z]{12}$</string>
+			<key>pfm_name</key>
+			<string>PayloadUUID</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload UUID</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<integer>1</integer>
+			<key>pfm_description</key>
+			<string>The version of the whole configuration profile.</string>
+			<key>pfm_description_reference</key>
+			<string>The version number of the individual payload.
+A profile can consist of payloads with different version numbers. For example, changes to the VPN software in iOS might introduce a new payload version to support additional features, but Mail payload versions would not necessarily change in the same release.</string>
+			<key>pfm_name</key>
+			<string>PayloadVersion</string>
+			<key>pfm_require</key>
+			<string>always</string>
+			<key>pfm_title</key>
+			<string>Payload Version</string>
+			<key>pfm_type</key>
+			<string>integer</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>This value describes the issuing organization of the profile, as displayed to the user</string>
+			<key>pfm_name</key>
+			<string>PayloadOrganization</string>
+			<key>pfm_title</key>
+			<string>Payload Organization</string>
+			<key>pfm_type</key>
+			<string>string</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Sets the network address for first run</string>
+			<key>pfm_name</key>
+			<string>network</string>
+			<key>pfm_title</key>
+			<string>Network Subdomain</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_value_placeholder</key>
+			<string>autoco</string>
+			<key>pfm_documentation_url</key>
+			<string>https://www.twingate.com/docs/macos-and-ios/#twingate-network-pre-configuration</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Starts the Twingate Client upon user login</string>
+			<key>pfm_name</key>
+			<string>startAtLogin</string>
+			<key>pfm_title</key>
+			<string>Start At Login</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+		</dict>
+		<dict>
+			<key>pfm_default</key>
+			<false/>
+			<key>pfm_description</key>
+			<string>Suppresses the first run Data Privacy screen</string>
+			<key>pfm_name</key>
+			<string>PresentedDataPrivacy</string>
+			<key>pfm_title</key>
+			<string>Suppress Data Privacy Screen</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
+			<key>pfm_documentation_url</key>
+			<string>https://www.twingate.com/privacy/</string>
```

Comment:
I don't see anything at this URL related to the `PresentedDataPrivacy` key. My thought would be to remove the URL if it isn't specifically taking us to documentation of the key.

Reply:
My thinking was to link the admin to the privacy policy since they are skipping/agreeing it on behalf of their users, but I see how that can be overkill and not add direct value for the admin creating the profile. Will remove.